.github/workflows/integration_tests.yml

```diff
@@ -23,28 +29,166 @@ on:
         type: string
         required: false
         description: 'Target machine (e.g., H1-1E).'
+      cudaq_test_image:
+        type: string
+        required: false
+        default: 'nvcr.io/nvidia/nightly/cuda-quantum:latest'
+        description: 'CUDA Quantum image to run the tests in. Default to the latest CUDA Quantum nightly image'
+      commit_sha:
+        type: string
+        required: false
+        description: 'Commit SHA to pull the code (examples/tests) for testing. Default to the commit associated with the CUDA Quantum docker image if left blank'
+      cudaq_nvcf_deploy_image:
+        type: string
+        required: false
+        default: 'nvcr.io/nvidia/nightly/cuda-quantum:latest'
+        description: 'CUDA Quantum image to use for NVCF deployment. Default to the latest CUDA Quantum nightly image'
+      workflow_id:
+        type: string
+        required: false
+        description: 'Workflow Id to retrieve the Python wheel for testing. Default to the wheels produced by the Publishing workflow associated with the latest nightly CUDA Quantum Docker image if left blank'
+
   schedule:
     - cron: 0 3 * * *
 
+env:
+  NVCF_FUNCTION_ID: 3bfa0342-7d2a-4f1b-8e81-b6608d28ca7d
```

Comment:
Will this be a unique function ID? With our default client behavior of "use the latest version", we don't want these tests to interfere with others' usage. I'm also wondering if this should be moved to a secret.

Reply:
We'll reserve/use this function Id for CI and create new functions for other usage, such as development/staging/production in the future. 
Re: repo secret vs. hardcoded value. There is no real reason to hide this UUID and keeping it in the code will allow updates via a regular PR workflow.